.github/workflows/update-github.yaml

```diff
@@ -0,0 +1,45 @@
+name: "UpdateOctokit"
+
+on:
+  schedule:
+    - cron: '0 18 * * 0'
+
+jobs:
+  UpdateOctokit:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Update Octokit
+      working-directory: packages/github 
+      run: |
+        npx npm-check-updates -u --dep prod
```

Comment:
- consider `--upgrade` instead of `-u` (long form for readability in scripts, short form for interactive terminal)
- is it fragile to assume `npm-check-updates` is installed globally?
- curious whether considered running npm update commands instead?

Reply:
- Great Idea, I'll update it
- npx will install items if they don't [exist in path](https://www.npmjs.com/package/npx)
- Update respect semver, which is nice, but ` npm-check-updates` will create a new pr will notify for new major versions.  Even if the PR isn't perfect (the new version isn't compatible), the pr will act as a notification that we need to do something!